consensus/src/round_manager.rs

```diff
@@ -580,6 +610,9 @@ impl RoundManager {
 
     fn sync_only(&self) -> bool {
         let sync_or_not = self.local_config.sync_only || self.block_store.vote_back_pressure();
+        if self.block_store.vote_back_pressure() {
+            warn!("Vote back pressure is set");
```

Comment:
Why should back pressure be a warning?

Reply:
Vote backpressure is when the difference between ordered round and committed round exceeds 7. If there is some problem with syncing or execution, it's good to know about it from logs.